### PR TITLE
update favicon and to svg

### PR DESCRIPTION
**Pull request by schwesig**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "gatsby": "4.20.0",
     "gatsby-plugin-catch-links": "4.20.0",
     "gatsby-plugin-google-gtag": "4.23.0",
-    "gatsby-plugin-manifest": "4.20.0",
+    "gatsby-plugin-manifest": "^4.20.0",
```
Comment (tumido) [inline package.json:29]: What is this change about? I think renovate will complain about the caret if this gets merged.

https://docs.renovatebot.com/dependency-pinning/

Comment (schwesig) [reply package.json:29]: thanks, good point. I trusted the npm installer from gatsby.
didn't think about renovate.
will remove

Comment (schwesig) [reply package.json]: done

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**